src/components/profile/MyAllegiances.js

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
```

Comment:
is this component necessary? the MyGroupAllegiances component (and some of the redux) was set up so the MyGroupsAllegiances component could just take prop type allegiances and work the same way.

Reply:
Solid point, I could definitely refactor those into a single component without too much trouble.

Reply:
Follow-up: were you referring to the GroupAllegiances component or to MyAllegianceGroups?